locales/esES.lua

```diff
@@ -0,0 +1,19 @@
+-- esES localization
+local L = LibStub("AceLocale-3.0"):NewLocale("WOWUP", "esES", true, true)
```

Comment:
The „, true, true“ needs to be removed. Otherwise this is the default language which will be used if there is no client local file.